x/accounts/defaults/lockup/continuous_locking_account.go

```diff
@@ -41,7 +41,7 @@ func (cva ContinuousLockingAccount) Init(ctx context.Context, msg *lockuptypes.M
 		return nil, sdkerrors.ErrInvalidRequest.Wrapf("invalid end time %s", msg.EndTime.String())
 	}
 
-	if msg.EndTime.Before(msg.StartTime) {
+	if msg.EndTime.Before(msg.StartTime) || msg.EndTime.Equal(msg.StartTime) {
```

Comment:
I think it would be more readable if we do, but it's just a nit:
`if !msg.EndTime.After(msg.StartTime) {`